tools/remote_test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e
+#
+# Copyright 2018 The Kythe Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Tests the Kythe repository using experimental Remote Build Execution.
+
```

Comment:
Can you add a bit more text here, discussing how the test tests remote execution, and what it means if it fails?

Reply:
K.